pinot-common/src/main/java/com/linkedin/pinot/common/utils/SegmentNameBuilder.java

```diff
@@ -78,9 +78,9 @@ public static String buildLowLevelConsumerSegmentName(String tableName, String p
     }
 
     public static String extractTableName(String segmentId) {
```

Comment:
Thanks for doing this, makes it much more readable
A comment somewhere in the file that describes the Long HighLevel, ShortHighLevel and LowLevel nameing scheme will be wonderful, and make things even easier to understand.